docs/user/openstack/customization.md

```diff
@@ -150,9 +150,17 @@ platform:
 
 ## Custom Subnets
 
-Before running the installer with a custom `machinesSubnet`, there are a few things you should check. First, make sure that you have a network and subnet available. Note that the installer will create ports on this network, and some ports will be given a fixed IP addresses in the subnet's CIDR. Make sure that the credentials that you give the installer are authorized to do this! The installer will also apply a tag to the subnet. Lastly, the installer will not create a router, so make sure to do this if you need a router to attach floating IP addresses to the network you want to install the nodes onto. Note that for now, this feature does not fully support provider networks.
+In the `install-config.yaml`, the `machinesSubnet` is the subnet where the Kubernetes endpoints of the nodes in your cluster will be published, and where the Ingress and API ports will be created. The installer creates a network and subnet for this purpose by default, however, you have the option to prevent the installer from creating those resources and overriding them with your own by setting the value of the `machinesSubnet` to the ID of an existing OpenStack subnet. To use this feature there are a few requirements:
```

Comment:
This is an improvement! I'd probably do something like this suggestion, but do what you like. 

```suggestion
In the `install-config.yaml` file, the value of the `machinesSubnet` property is the subnet where the Kubernetes endpoints of the nodes in your cluster are published.  The Ingress and API ports are created on this subnet, too.

By default, the installer creates a subnet for these endpoints and ports. Alternatively, you can use a subnet of your own by setting the value of the `machinesSubnet` property to the ID of an existing OpenStack subnet. 

To use your own subnet, your cluster must meet the following requirements:
```

Thoughts:

- Do we need to indicate "network and subnet," here, or is just "subnet" sufficient?
- Use "ID" or "UUID?"

Reply:
It creates both a network and subnet, so its good to be clear about that. UUID might be better